examples/prometheus/README.md

```diff
@@ -5,9 +5,28 @@ This template creates a Prometheus instance preconfigured to gather OpenShift an
 To deploy, run:
 
 ```
-$ oc new-app -f prometheus.yaml
+$ oc new-app -f https://raw.githubusercontent.com/openshift/origin/master/examples/prometheus/prometheus.yaml
 ```
 
+This template sets up an oauth proxy for authentication.  The oauth-proxy relies on the same authentication system configured for your OpenShift cluster.
+
+See [this documentation](https://docs.openshift.org/latest/install_config/configuring_authentication.html) for authentication options for OpenShift.
+
+One option documented above is htpasswd.  Create an htpasswd user called (for example) ```prometheus```.
+
+The ```prometheus``` user must have cluster-reader access.  Grant the user cluster-reader access:
```

Comment:
Actually only needs view access on the namespace that has the route.  Would recommend `oc policy add-role-to-user view -n kube-system prometheus`

Reply:
OK, I thought I had seen it here:  https://github.com/openshift/openshift-ansible/blob/master/roles/openshift_prometheus/tasks/install_prometheus.yaml#L48

I'll test out your suggestion to double check that things still work.

Reply:
That does work.

Reply:
Would be nice to include this in the readme for openshift-prometheus. I'll send a PR.